examples/peripherals/i2s_adc_dac/sdkconfig.default

```diff
@@ -7,7 +7,7 @@
 # SDK tool configuration
 #
 CONFIG_TOOLPREFIX="xtensa-esp32-elf-"
-CONFIG_PYTHON="python"
```

Comment:
I'd rather not change this default here. macOS, and some Linux distros, still ship with `python` and no `python3`.